TransferHelperApp/src/app/player-api.service.ts

```diff
@@ -0,0 +1,41 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PlayerApiService {
+
+  readonly playerAPIUrl = "https://localhost:7228/api";
```

Comment:
z tego co widzę używasz tego `playerAPIUrl` ogólnie do połączeń, nie tylko do obsługi endpointów obecnych w _PlayerControllerze_, więc widziałbym raczej to jako: `BaseApiUrl`

Reply:
Zmieniono nazwę pliku na api.service.ts oraz zmieniono nazwę url na `baseApiUrl`